vignettes/parallel.Rmd

```diff
@@ -144,6 +145,34 @@ perf_boxplot +
 You can also create your own plots however you like using the performance
 results.
 
+## Live progress updates
+
+`preprocess_data()` and `get_feature_importance()` support reporting live
+progress updates using the `progressr` package. The format is up to you, but we
+recommend using a progress bar like this:
+
+```{r progress, eval = FALSE}
+# optionally, specify the progress bar format with the `progress` package
+progressr::handlers(progressr::handler_progress(
+    format = ":message :bar :percent | elapsed: :elapsed | eta: :eta",
+    clear = FALSE,
+    show_after = 0))
+# tell progressr to always report progress in any functions that use it
+progressr::handlers(global = TRUE)
+
+# run your code and watch the live progress updates
+dat <- preprocess_data(otu_mini_bin, 'dx')$dat_transformed
+#> Using 'dx' as the outcome column.
+#> preprocessing ========================>-------  78% | elapsed:  1s | eta:  0s
+results <- run_ml(dat, "glmnet", kfold = 2, cv_times = 2,
+                  find_feature_importance = TRUE)
+#> Using 'dx' as the outcome column.
+#> Training the model...
+#> Training complete.
+#> Feature importance =========================== 100% | elapsed: 37s | eta:  0s
+```
+
+
```

Comment:
Thoughts on adding a line that says how to remove the progress bar if you decide you don't want it?

```
progressr::handlers(global = FALSE)
```

Reply:
I encourage them to read the `progressr` docs instead: https://github.com/SchlossLab/mikropml/pull/257/commits/9e2dc5f1e225c0556a5edf0432083d9fffdade1d